packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManagerImpl.kt

```diff
@@ -286,4 +286,13 @@ internal object LottieAnimationViewManagerImpl {
     ) {
         viewManager.textFilters = textFilters
     }
+
+    @JvmStatic
+    fun setSourceDotLottieURI(
+        uri: String?,
+        viewManager: LottieAnimationViewPropertyManager
+    ) {
+        viewManager.sourceDotLottie = uri
```

Comment:
Considering what react bridge does when a field goes from string to undefined on iOS, do you think we should also add the clean up here as well for Android?

Reply:
Nit: Also please move this up under line 168 so all the source settings are in the same place

Reply:
For Android we have null check everywhere `LottieAnimationViewPropertyManager`. I moving it up.